src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue

```diff
@@ -0,0 +1,230 @@
+<!--
+  - @copyright Copyright (c) 2023 Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @author Marco Ambrosini <marcoambrosini@icloud.com>
+  - @author Dorra Jaouad <dorra.jaoued7@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<!-- reactions buttons and popover with details -->
+	<div class="reactions-wrapper">
+		<NcPopover v-for="reaction in Object.keys(detailedReactions)"
+			:key="reaction"
+			:delay="200"
+			:focus-trap="false"
+			:triggers="['hover']">
```

Comment:
```vue
@hover="reactionsStore.fetchReactions"
```

Maybe that would be enough?

Reply:
As the count is now aggregated from reactions store, fetching is mandatory regardless of hover status, otherwise user can't see the reactions until they hover.

Reply:
My intention was to keep both plain and detailed reactions in the store independently. Thus, general overview is always available, and if you want to see details, you simply make a specific request for this

Reply:
So, as an overview : 

- Plain reactions ( aka reactions and their counts) are stored in Message and can be passed as a prop. This to always show up to date reactions count but only fetch details when you hover. A drawback to this way is that the count and the details are synchronized ( you see count outdated but the details are updated because of the recent fetch). However, I suggest a solution for it: Whenever there is a fetch on hover, we can replace the count coming from Message by the calculated length of detailed reactions.

-  In case we still want to get rid of the dependency via props passed from Message component. As I explained somewhere, we use the data from the quote reply of adding a reaction so we don't need to trigger any fetch requests. However, there will be a fetch request for each reaction revoke but there won't be many. Count is calculated by length in a computed prop and we don't need to store it. 

IMO, we can go with the second option.